src/pages/blog/blog-landing.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
@Auhseh 
Please revert any changes unrelated to the header component please, including any static assets.  Those should be done as part of another issue / PR (which I plan to work on initially)

Reply:
Okay, done